DashAI/back/dependencies/database/models.py

```diff
@@ -100,3 +100,41 @@ def set_status_as_finished(self) -> None:
     def set_status_as_error(self) -> None:
         """Update the status of the run to error."""
         self.status = RunStatus.ERROR
+
+
+class GlobalExplainer(Base):
+    __tablename__ = "global_explainer"
+    """
+    Table to store all the information about a global explainer.
+    """
+    id: Mapped[int] = mapped_column(primary_key=True)
+    name: Mapped[str] = mapped_column(String, unique=True, nullable=False)
+    run_id: Mapped[int] = mapped_column(nullable=False)
+    explainer_name: Mapped[str] = mapped_column(String, nullable=False)
+    explanation_path: Mapped[str] = mapped_column(String, nullable=True)
+    plot_path: Mapped[str] = mapped_column(String, nullable=True)
```

Comment:
It could be that some global explainer has several plots? Or this is just the path to the folder where te plots will be stored? (in that case, i suggest to rename the variable to plots_path).

Reply:
only for local explainers, so i'll rename it